### PR TITLE
fix(ras-acc): merge trunk into epic/ras-acc

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR is to merge trunk into the epic/ras-acc branch after the dependency updates.

### How to test the changes in this Pull Request:

1. Run `npm run build` and confirm there are no errors.
2. Spot check the updates and confirm that nothing looks out of place.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: includes/class-modal-checkout.php [hunk 1]
```diff
@@ -1478,7 +1478,7 @@ public static function get_summary_card_price_string( $name, $price = '', $frequ
 	 *
 	 * @return void.
 	 */
-	public static function maybe_add_checkout_registration_order_meta( $order ) {
+	public static function maybe_add_checkout_registration_order_meta( $order ) { // phpcs:ignore WordPressVIPMinimum.Hooks.AlwaysReturnInFilter.VoidReturn, WordPressVIPMinimum.Hooks.AlwaysReturnInFilter.MissingReturnStatement
```
Comment (laurelfulford) [inline includes/class-modal-checkout.php]: I wanted to flag this since I'm not sure ignoring these sniffs is the preferred way to fix these errors.

**Comment by laurelfulford** [comment]
@dkoo Roping you in on this one, too 🙂 Just let me know if I should be looping in anyone else instead!

File: includes/class-modal-checkout.php (hunk 1 above)
Comment (dkoo) [reply includes/class-modal-checkout.php]: It looks like this hook is actually an [action, not a filter](https://woocommerce.github.io/code-reference/files/woocommerce-includes-class-wc-checkout.html#source-view.449). Updated the `add_action` call in 4399227 and that means we can remove the `// phpcs:ignore` statement as well.

**Review by dkoo — APPROVED** [review]
@laurelfulford the changes look and work well for me, so if 4399227 looks good to you, this can be merged.

**Comment by laurelfulford** [comment]
Ah, thanks for catching that, @dkoo! I wrongly assumed that the sniff was being to sensitive since this wasn't an issue before.

This change looks good to me, so I think this is good to go!

**Pull request merged**